include/boost/geometry/algorithms/detail/overlay/get_turn_info_helpers.hpp

```diff
@@ -439,22 +429,21 @@ class intersection_info
     typedef typename base::side_calculator_type side_calculator_type;
     typedef typename base::result_type result_type;
     
-    typedef typename boost::tuples::element<0, result_type>::type i_info_type; // intersection_info
-    typedef typename boost::tuples::element<1, result_type>::type d_info_type; // dir_info
+    typedef typename result_type::intersection_points_type i_info_type; // intersection_info
+    typedef typename result_type::direction_type d_info_type;           // dir_info
```

Comment:
The comments can be omitted now, the types are clear enough.

Reply:
Right.